### PR TITLE
UI Elements

### LINKED ISSUE #8412: NoSuchKeyException: backgroundColor app crash when tap on 'Network Fee' (android)
***Type***: Bug

***Summary***: With the new wallet sign in transaction flow (https://github.com/status-im/status-react/pull/8361) every time I'm trying to modify 'Network Fee' on Android build - the 
`com.facebook.react.bridge.NoSuchKeyException: backgroundColor` error appears followed by app crash when tap on 'Network fee' element from bottom sheet.

#### Expected behavior

Network Fee sheet appears
#### Actual behavior
- `com.facebook.react.bridge.NoSuchKeyException: backgroundColor` error appears followed by app crash on Android.
Screenshot 2019-08-14 at 12 58 14

### Reproduction 

- Open Status and create new account
- Navigate to `Wallet` -> `Status account` -> `Send`
- Specify valid address in `Specify recipient...` field and enter `0` in `Amount` field
- Tap `Sign transaction`
- Tap `Network fee`
 **Actual result on Android:** `com.facebook.react.bridge.NoSuchKeyException: backgroundColor` error followed by app crash when tap `Exit` button in pop-up

### Additional Information

- Status version: Develop 0.14.0 (2019061312) 
- Operating System: Android and iOS

#### Logs

```
--------- beginning of main
06-13 15:20:31.062 29800 29800 D ViewRootImpl@46f0e4f[MainActivity]: ViewPostIme pointer 0
06-13 15:20:31.220 29800 29800 D ViewRootImpl@46f0e4f[MainActivity]: ViewPostIme pointer 1
06-13 15:20:31.246 29800 29854 D ReactNativeJS: DEBUG [status-im.utils.handlers:34] - Handling re-frame event:  :signing.ui/open-fee-sheet
--------- beginning of crash
06-13 15:20:31.498 29800 29855 E AndroidRuntime: FATAL EXCEPTION: mqt_native_modules
06-13 15:20:31.498 29800 29855 E AndroidRuntime: Process: im.status.ethereum.pr, PID: 29800
06-13 15:20:31.498 29800 29855 E AndroidRuntime: com.facebook.react.bridge.NoSuchKeyException: backgroundColor
06-13 15:20:31.498 29800 29855 E AndroidRuntime: 	at com.facebook.react.bridge.ReadableNativeMap.getValue(ReadableNativeMap.java:124)
06-13 15:20:31.498 29800 29855 E AndroidRuntime: 	at com.facebook.react.bridge.ReadableNativeMap.getValue(ReadableNativeMap.java:128)
06-13 15:20:31.498 29800 29855 E AndroidRuntime: 	at com.facebook.react.bridge.ReadableNativeMap.getInt(ReadableNativeMap.java:182)
06-13 15:20:31.498 29800 29855 E AndroidRuntime: 	at com.facebook.react.uimanager.ViewProps.isLayoutOnly(ViewProps.java:230)
06-13 15:20:31.498 29800 29855 E AndroidRuntime: 	at com.facebook.react.uimanager.NativeViewHierarchyOptimizer.isLayoutOnlyAndCollapsable(NativeViewHierarchyOptimizer.java:445)
06-13 15:20:31.498 29800 29855 E AndroidRuntime: 	at com.facebook.react.uimanager.NativeViewHierarchyOptimizer.handleCreateView(NativeViewHierarchyOptimizer.java:92)
06-13 15:20:31.498 29800 29855 E AndroidRuntime: 	at com.facebook.react.uimanager.UIImplementation.handleCreateView(UIImplementation.java:306)
06-13 15:20:31.498 29800 29855 E AndroidRuntime: 	at com.facebook.react.uimanager.UIImplementation.createView(UIImplementation.java:298)
06-13 15:20:31.498 29800 29855 E AndroidRuntime: 	at com.facebook.react.uimanager.UIManagerModule.createView(UIManagerModule.java:462)
06-13 15:20:31.498 29800 29855 E AndroidRuntime: 	at java.lang.reflect.Method.invoke(Native Method)
06-13 15:20:31.498 29800 29855 E AndroidRuntime: 	at com.facebook.react.bridge.JavaMethodWrapper.invoke(JavaMethodWrapper.java:372)
06-13 15:20:31.498 29800 29855 E AndroidRuntime: 	at com.facebook.react.bridge.JavaModuleWrapper.invoke(JavaModuleWrapper.java:158)
06-13 15:20:31.498 29800 29855 E AndroidRuntime: 	at com.facebook.react.bridge.queue.NativeRunnable.run(Native Method)
06-13 15:20:31.498 29800 29855 E AndroidRuntime: 	at android.os.Handler.handleCallback(Handler.java:790)
06-13 15:20:31.498 29800 29855 E AndroidRuntime: 	at android.os.Handler.dispatchMessage(Handler.java:99)
06-13 15:20:31.498 29800 29855 E AndroidRuntime: 	at com.facebook.react.bridge.queue.MessageQueueThreadHandler.dispatchMessage(MessageQueueThreadHandler.java:29)
06-13 15:20:31.498 29800 29855 E AndroidRuntime: 	at android.os.Looper.loop(Looper.java:164)
06-13 15:20:31.498 29800 29855 E AndroidRuntime: 	at com.facebook.react.bridge.queue.MessageQueueThreadImpl$4.run(MessageQueueThreadImpl.java:232)
06-13 15:20:31.498 29800 29855 E AndroidRuntime: 	at java.lang.Thread.run(Thread.java:764)

```

### PR DESCRIPTION
**Pull request by flexsurfer**
**Fixed**

fixes #8412

**Implemented**

UI elements: button (main, secondary, next, previous), badge 

image

Toolbar

image

**Removed**

- action button
- button (legacy)
- bottom buttons (legacy)
- list header
- cell

**QA**
review all UI elements and screens

### CODE REVIEW

File: components/src/status_im/ui/components/react.cljs [hunk 1]
```diff
@@ -298,39 +298,7 @@
 
 (defmethod create-main-screen-view :iphone-x [current-view]
   (fn [props & children]
-    (let [props             (merge props
```
Comment (yenda) [inline components/src/status_im/ui/components/react.cljs:304]: where did this go?

Comment (flexsurfer) [reply components/src/status_im/ui/components/react.cljs:304]: we don't need this anymore

**Comment by asemiankevich** [comment]
@flexsurfer there is an error when editing user picture:

1. create new account
2. profile
3. tap edit picture

image

**Comment by asemiankevich** [comment]
+ additionally:

1. update fee is broken

image


2. Accept invitation button is gone

image


3. Sign transaction button is gone

image

**Comment by flexsurfer** [comment]
@asemiankevich all fixed, thanks!

**Review by dmitryn — APPROVED** [review]

**Comment by asemiankevich** [comment]
found one more issue:

image


happens when:
1. i have new account
2. i faucet some funds and stt
3. i added additional account
4. i navigate to status account and tap Send


I can't reproduce in nightly


@flexsurfer we have it but in another place as well https://github.com/status-im/status-react/issues/8412.

**Comment by flexsurfer** [comment]
@asemiankevich fixed, thanks!

**Comment by asemiankevich** [comment]
@flexsurfer this screen supposed to have blue background i think
image

**Pull request merged**